data/helpers.d/backend

```diff
@@ -51,3 +51,41 @@ ynh_remove_logrotate () {
 		sudo rm "/etc/logrotate.d/$app"
 	fi
 }
+
+# Create a dedicated nginx config
+#
+# usage: ynh_add_nginx_config
+ynh_add_nginx_config () {
+	finalnginxconf="/etc/nginx/conf.d/$domain.d/$app.conf"
+	ynh_compare_checksum_config "$finalnginxconf" 1
```

Comment:
Uh, it's not clear for me what this line is intended to do ? Is it meant to backup the file for, like, if you runned ynh_add_nginx_config twice for instance ?